Pod/Classes/VOKManagedObjectMap.m

```diff
@@ -57,6 +57,7 @@ + (NSDateFormatter *)vok_defaultDateFormatter
         DefaultDateFormatter = [NSDateFormatter new];
         DefaultDateFormatter.dateFormat = @"yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'";
         DefaultDateFormatter.timeZone = [NSTimeZone timeZoneForSecondsFromGMT:0];
+        DefaultDateFormatter.locale = [NSLocale localeWithLocaleIdentifier:@"en_US_POSIX"];
```

Comment:
Would it make sense to make a constant out of this string literal in the .h file? That way it can be shared here and in tests?